process/sync/baseSync.go

```diff
@@ -197,6 +196,49 @@ func (boot *baseBootstrap) applyBlock(
 	return nil
 }
 
+func (boot *baseBootstrap) addHeadersToForkDetector(
+	shardId uint32,
+	nonce uint64,
+	getHeader func(uint32, uint64) (data.HeaderHandler, []byte, error)) {
```

Comment:
add enter before the return variables.

Reply:
Removed method as parameter after implemented an interface